otmonitor.vfs/otmonitor.tcl

```diff
@@ -2136,6 +2139,12 @@ register 82	{f8.8}		"Exhaust inlet temperature"
 register 83	{f8.8}		"Exhaust outlet temperature"
 register 84	{u16}		"Exhaust fan speed"
 register 85	{u16}		"Inlet fan speed"
+register 98	{f8.8}		"For a specific RF sensor the RF strength and battery level is written" ;# Data-type assumed
+register 99	{f8.8}		"Operating Mode HC1, HC2/ Operating Mode DHW"				;# Data-type assumed
+register 109	{u16}		"Electricity producer starts"
+register 110	{u16}		"Electricity producer hours"
+register 111	{u16}		"Electricity production"
+register 112	{u16}		"Cumulativ Electricity production"
```

Comment:
"Cumulative" is the correct spelling. And I see no reason to capitalize "electricity". Same for "humidity" and "mode" in msgids 38 and 99.